gophernicus.1

```diff
@@ -0,0 +1,251 @@
+Select the output charset. It can be
+.Ar UTF-8 ,
+.Ar US-ASCII ,
+or
+.Ar ISO-9959-1 .
+The default is
+.Ar UTF-8 .
+.It Fl s Ar seconds
+Session timeout in seconds.
+The default is
+.Pa 1800 .
+.It Fl i Ar hits
+Maximum hits until throttling.
+The default is
+.Pa 4069 .
+.It Fl k Ar kilobytes
+Maximum transfer size in KiB until throttling.
+The default is
+.Pa 4194304
+(4 GiB).
+.It Fl f Ar directory
+Set directory where output filters are found.
+Disabled by default.
+.It Fl e Ar ext Ns = Ns Ar type
+Map file extension
+.Ar ext
+to gopher filetype
+.Ar type .
+.It Fl R Ar old Ns = Ns Ar new
+Rewrite the start of a selector.
+.It Fl D Ar description
+Set server description to appear in
+.Pa /caps.txt .
+If
+.Ar description
+looks like a filename (starts with a /), use the contents of the file.
+.It Fl L Ar location
+Set server location to appear in
+.Pa /caps.txt .
+If
+.Ar location
+looks like a filename (starts with a /), use the contents of the file.
+.It Fl A Ar email
+Set the email of the administrator to appear in
+.Pa /caps.txt .
+.It Fl U Ar paths
+Set a colon-separated list of extra
+.Xr unveil 2
+paths (only for OpenBSD).
+.It Fl nv
+Disable virtual hosting.
+.It Fl nl
+Disable parent directory links.
+.It Fl nh
+Disable menu header.
+.It Fl nf
+Disable menu footer.
+.It Fl nd
+Don't show date and size of files in menu.
+.It Fl nc
+Disable file content detection (similar to
+.Xr magic 5 Ns ).
+.It Fl no
+Disable output charset conversion.
+.It Fl nq
+Disable HTTP-style query strings
+.Ql ?foo=bar&baz=quux .
+.It Fl ns
+Disable logging to
+.Xr syslog 3 .
+.It Fl na
+Disable autogenerated
+.Pa /caps.txt .
+.It Fl nt
+Disable
+.Pa /server-status .
+.It Fl nm
+Disable shared memory use (for debugging purposes).
+.It Fl nr
+Disable root user check (for debugging purposes).
+By default,
+.Nm
+will refuse to run as root.
+.It Fl np
+Disable HAProxy proxy protocol.
+.It Fl nx
+Disable execution of gophermaps and scripts.
+.It Fl nH
+Disable HTTP response to HTTP GET and POST requests.
+.It Fl d
+Print debug output in
+.Xr syslog 3
+and
+.Pa /server-status .
+.It Fl v
+Display version information and build date.
+.It Fl b
+Display the BSD license.
+.It Fl \&?
+Display this help.
+.El
+.Pp
+Default values may have been changed at compile time by
+.Pa gophernicus.env .
+.Sh STANDARDS
+.Nm
+mostly conforms to
+.Lk https://tools.ietf.org/html/rfc1436 "RFC 1436"
+.Sh AUTHORS
+.An -nosplit
+.An Kim Holviala.
+wrote the original implementation (2009\(en2018).
+.Pp
+.An fosslinux
+is the Current maintainer (2019\(en).
+.Pp
+Code contributed by
+.An hb9kns
+and others.
+.Pp
+Developpers can be reached at
+.Mt gophernicus.AT.gophernicus.DOT.org
+.Sh BUGS
+Known bugs are listed as issues on
+.Nm Ap s GitHub page:
+.Lk https://github.com/gophernicus/gophernicus/issues
+.Pp
+Please report any bug you might experience there as well.
+.Sh COPYRIGHT
+Copyright \(co
+.An Kim Holviala
+2009\(en2018.
+.Pp
+Copyright \(co
+.An Nm gophernicus Ns \& developpers
```

Comment:
developpers typo

Reply:
Thanks, I always get tripped by this one…